docs/template-windows.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: windows-vm-${NAME}
+  spec:
+    template:
+      metadata:
+      labels:
```

Comment:
labels need to be indentet a little more, but as emtpy we can remove metadat alltogether